agent/utils/ticker_test.go

```diff
@@ -26,7 +26,8 @@ const (
 )
 
 func TestTickerHappyCase(t *testing.T) {
-	ctx, _ := context.WithTimeout(context.Background(), 1000*time.Millisecond)
```

Comment:
`agent/utils/ticker_test.go:29: the cancel function returned by context.WithTimeout should be called, not discarded, to avoid a context leak`